config/crd/bases/pulpproject_v1beta1_pulp_crd.yaml

```diff
@@ -176,9 +176,17 @@ spec:
               object_storage_azure_secret:
                 description: The secret for Azure blob storage configuration.
                 type: string
+              gunicorn_timeout:
```

Comment:
can you help me understand what for and where this yaml  file is used?
should you also add here number of pulp-api and pulp-content workers?

https://github.com/pulp/pulp-operator/commit/bd49ac95d1b7361c75aa1b526ce1ff4cf6dfd310

Reply:
this is for avoiding the experience you had when using `default_settings`, here we declare the variables the Pulp CR will accept, if it is not declared here, it gets cleaned

Reply:
and yes, I should add those vars